[2024-09-24T19:06:18.984+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-24T19:06:19.014+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1.task2 scheduled__2024-09-24T00:14:00+00:00 [queued]>
[2024-09-24T19:06:19.034+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1.task2 scheduled__2024-09-24T00:14:00+00:00 [queued]>
[2024-09-24T19:06:19.035+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-09-24T19:06:19.060+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task2> on 2024-09-24 00:14:00+00:00
[2024-09-24T19:06:19.076+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=108) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-24T19:06:19.078+0000] {standard_task_runner.py:72} INFO - Started process 109 to run task
[2024-09-24T19:06:19.079+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'task2', 'scheduled__2024-09-24T00:14:00+00:00', '--job-id', '575', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpvzvhmf_3']
[2024-09-24T19:06:19.080+0000] {standard_task_runner.py:105} INFO - Job 575: Subtask task2
[2024-09-24T19:06:19.151+0000] {task_command.py:467} INFO - Running <TaskInstance: dag1.task2 scheduled__2024-09-24T00:14:00+00:00 [running]> on host 5d6b20a950c7
[2024-09-24T19:06:19.247+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etl_user' AIRFLOW_CTX_DAG_ID='dag1' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T00:14:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-24T00:14:00+00:00'
[2024-09-24T19:06:19.248+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-24T19:06:19.267+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-24T19:06:19.268+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello, World!2"']
[2024-09-24T19:06:19.283+0000] {subprocess.py:86} INFO - Output:
[2024-09-24T19:06:19.284+0000] {subprocess.py:93} INFO - Hello, World!2
[2024-09-24T19:06:19.285+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-24T19:06:19.314+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-24T19:06:19.315+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=task2, run_id=scheduled__2024-09-24T00:14:00+00:00, execution_date=20240924T001400, start_date=20240924T190619, end_date=20240924T190619
[2024-09-24T19:06:19.376+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-24T19:06:19.398+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-24T19:06:19.401+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
